bindings/jni/src/main/java/org/zeromq/czmq/Zconfig.java

```diff
@@ -238,11 +238,9 @@ public void removeSubtree () {
     }
     /*
     Destroy node and subtree (all children)
-    WARNING: manually fixed void -> long
```

Comment:
please revert the changes to this file, see https://github.com/zeromq/czmq/commit/290f0129f20732847ef496b7758f910e18981d39

(yes it's very annoying)